Specs/Data/Tiles3D/instanced/instancedGltfEmbedded/tiles.json

```diff
@@ -0,0 +1 @@
+{"properties":{"id":{"minimum":0,"maximum":99},"height":{"minimum":0.7289306027814746,"maximum":97.26047022268176}},"geometricError":40,"root":{"box":[-1.319970476767974,0.6986942109000001,-1.3199295232320258,0.6987257891,0,15],"geometricError":0,"content":{"url":"instancedGltfEmbedded.i3dm"}}}
```

Comment:
Not sure, but I can rename the folder to `Cesium3DTiles` for now. `Batched3DModel3DTileContentProvider` for example wouldn't be as clear as `Batched3DModelTiles3DContentProvider`. That name itself is too long anyways, and could also change.